### ISSUE TITLE
Publish binaries

### ISSUE THREAD

**Issue by nigredo-tori** [ritschwumm/xsbt-webstart#8]
Right now to use this plugin in a project one has to build it (and its dependencies) and install locally, or (in case of a team) to some private repository. This is ugly and painful - and raises the threshold for adoption.

Publishing to [bintray](https://www.scala-sbt.org/1.0/docs/Bintray-For-Plugins.html) or [Sonatype](https://www.scala-sbt.org/1.x/docs/Using-Sonatype.html) is relatively straightforward, and will save your users a lot of trouble.

**Comment by ritschwumm** [comment]
yeah, i've been delaying this for a long time. what do you think about jitpack?

**Comment by nigredo-tori** [comment]
There are a few possible issues concerning repository proxying. I'm using Nexus here as an example, since that's what I'm familiar with.

> jitpack

Haven't heard about it before. It looks nifty. However:
1. I'm not sure it supports directory indices, so it might require some extra configuration for Nexus.
2. Depending on how Nexus, Ivy and SBT (coursier?) handle timeouts and resolution failures, the first time the dependency is requested it might be presumed missing, and this result might be cached at proxy/user/project levels.

> using a github project as a maven repo

That sounds slightly more predictable. However:
1. It would require adding the respository to Nexus just for three libraries.
2. Directory listings are not supported.

Basically, I should be able to make either of those work in my case, but I would really like something more boring. SBT documentation recommends Bintray for publishing plugins, and provides a step-by-step instruction: https://www.scala-sbt.org/1.0/docs/Bintray-For-Plugins.html
